tools/github-actions/get-npm-tag/action.yml

```diff
@@ -22,5 +22,5 @@ outputs:
     description: 'NPM Tag to apply'
     value: ${{ steps.get-npm-tag.outputs.tag }}
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
why not `node18` ?

Reply:
because`node18` doesn't exist https://github.com/orgs/community/discussions/53217